cfgov/preprocessed/css/organisms/sidebar-breakout.less

```diff
@@ -0,0 +1,152 @@
+/* topdoc
+  name: Sidebar Breakout
+  family: cfgov-organisms
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="o-sidebar-breakout
+                    content__flush-bottom">
+            <div class="content_intro">
+                <div class="content-l">
+                    <section class="o-sidebar-breakout_col
+                                    content-l_col
+                                    content-l_col-2-3">
+                        <m-text-introduction></m-text-introduction>
```

Comment:
Ahh I gotcha. Hm, how about just the opening expected element since some organisms could include molecules that use difference opening elements:

`<div class="m-text-introduction">…</div>`